DFE-Digital/early-careers-framework
# Cpdrp 532 it materials link

### Context

Ticket: [532](https://dfedigital.atlassian.net/browse/CPDRP-532) 

### Changes proposed in this pull request

Adding a link to the induction tutor materials on the list cohorts page.

### Guidance to review

### Testing
Cypress tests were already in there. 

### Review Checks

### How can I view this in a review app?
- Login in as an induction tutor 
- Go to your cohorts. It should show you the text set out in the ticket with a link to the chosen cips induction materials page.
